tests/api/InclusionProofTest.ts

```diff
@@ -119,7 +119,9 @@ describe('InclusionProof', () => {
     const requestId = await RequestId.create(publicKey, authenticator.stateHash);
     const inclusionProof = new InclusionProof(merkleTreePath, authenticator, transactionHash);
 
-    expect(await inclusionProof.verify(requestId.toBigInt())).toEqual(InclusionProofVerificationStatus.OK);
+    expect(await inclusionProof.verify(requestId.toBitString().toBigInt())).toEqual(
+      InclusionProofVerificationStatus.OK,
```

Comment:
If I read the code correctly, this bigint approach of having 1 as first bit is dragged down to the merkle tree internals, and is also not documented